### PR TITLE
Add graphql resolver to get current user

### LINKED ISSUE #17: Implement stub getCurrentUser graphql query
We need a stub `getCurrentUser` query on the mock graphql schema. This will need to return a different user object depending on the token sent over so that the mock can be used to assume different roles for testing / development (we'll want to be able to sign in as staff or an author initially).   

Graphql schema for user TBD.

### PR DESCRIPTION
**Pull request by erezmus**
Implements a GraphQL resolver to return a dummy current user

Closes #17

### CODE REVIEW

File: src/mock-graphql/typeDefs.ts [hunk 1]
```diff
@@ -7,6 +7,12 @@ export const typeDefs = gql`
         updated: String!
     }
 
+    type User {
```
Comment (Cooryd) [inline src/mock-graphql/typeDefs.ts]: Do we want this to be a `Profile` type in anticipation of the mock `People` we'll need for the people picker query?

File: src/use-cases/getCurrentUser.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { Request, Response } from 'express';
+import currentUser from '../mock-data/currentUser.json';
```
Comment (Cooryd) [inline src/use-cases/getCurrentUser.ts]: We've just merged in the configurable data stuff. If you merge master into this and do something like 
```
import getMockData from '../getMockData';

export const getCurrentUser = () => (req: Request, res: Response): void => {
    const currentUser = getMockData('currentUser.json');
    res.json(currentUser);
};

```

File: src/mock-data/currentUser.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "id": "ewwboc7m",
```
Comment (Cooryd) [inline src/mock-data/currentUser.json]: We've moved mock-data out of src and into the top level directory as part of configurable data

File: src/mock-graphql/resolvers.ts [hunk 4]
```diff
@@ -1,9 +1,10 @@
-import { getSubmissions, startSubmission } from '../use-cases';
+import { getSubmissions, startSubmission, getCurrentUser } from '../use-cases';
 
 const submissions = [];
 
 export const resolvers = {
     Query: {
+        getCurrentUser: getCurrentUser(),
```
Comment (diversemix) [inline src/mock-graphql/resolvers.ts]: Let's call this `getCurrentProfile`

Comment (erezmus) [reply src/mock-graphql/resolvers.ts]: done. should we also change the graphql type from User to Profile?

**Pull request merged**